### PR TITLE
add docker-compose setup for easier local dev

### PR DESCRIPTION
**Pull request by hwsoderlund**
This allows us to `docker-compose up` to get a local fillpdf service in a docker container.

### CODE REVIEW

**Comment by per-heed-md** [comment]
Still the same error

```
 fillpdf > docker-compose up
[+] Building 2.7s (13/15)
 => [internal] load build definition from Dockerfile                   0.0s
 => => transferring dockerfile: 963B                                   0.0s
 => [internal] load .dockerignore                                      0.0s
 => => transferring context: 2B                                        0.0s
 => [internal] load metadata for docker.io/library/alpine:3.8          1.4s
 => [internal] load metadata for docker.io/library/golang:1.11-alpine  1.3s
 => [cached_base 1/2] FROM docker.io/library/golang:1.11-alpine3.8@sh  0.0s
 => [internal] load build context                                      0.4s
 => => transferring context: 12.05MB                                   0.4s
 => [stage-2 1/5] FROM docker.io/library/alpine:3.8@sha256:2bb501e617  0.0s
 => CACHED [stage-2 2/5] RUN apk add --no-cache ca-certificates tzdat  0.0s
 => [stage-2 3/5] RUN addgroup -S -g 101 mindoktor && adduser -u 100   0.3s
 => CACHED [cached_base 2/2] RUN apk add --no-cache ca-certificates t  0.0s
 => [build_base 1/3] WORKDIR /fillpdf                                  0.1s
 => [build_base 2/3] COPY . .                                          0.0s
 => ERROR [build_base 3/3] RUN go build -ldflags="-X main.version=$VE  0.8s
------
 > [build_base 3/3] RUN go build -ldflags="-X main.version=$VERSION":
#0 0.557 # net
#0 0.557 exec: "gcc": executable file not found in $PATH
------
failed to solve: executor failed running [/bin/sh -c go build -ldflags="-X main.version=$VERSION"]: exit code: 2
```

**Comment by hwsoderlund** [comment]
FYI, the issue was resolved after adding alpine-sdk to the Dockerfile.

**Comment by hwsoderlund** [comment]
Ready for review

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module github.com/mindoktor/fillpdf
+
+go 1.18
```
Comment (per-heed-md) [inline go.mod:3]: I guess it's no issue, but looks a bit odd to have Go 1.18 here and Go 1.11 in the Dockerfile

**Review by per-heed-md — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (hwsoderlund) [reply go.mod:3]: Oh, hmm, right, I didn't think of that. Yes, that is stupid. Go 1.11 will probably not care about the go.mod file, but it's still stupid. The only reason I added it was that go wouldn't even allow me to build and run without a go.mod file. I'll see if it's possible to just change the go version.

**Comment by hwsoderlund** [comment]
Totally forgot about this one. Merging now.

**Comment by phelian** [comment]
Not like you to forget @hwsoderlund ;)

**Comment by hwsoderlund** [comment]
I am an old man now. My memory fails me. How are things with you?

**Comment by phelian** [comment]
Things are good, currently in Switzerland (Did a massive mountain hike yesterday) 
Are you ever in malmö? lets do a lunch someday?

**Comment by hwsoderlund** [comment]
Switzerland. Sounds nice. I have 20 spänn waiting for you if you bring me one of those gigantic Toblerones. 

Sure, I am in Malmö from time to time. Once a week usually, except if I don't feel like it. :) Yes, let's do lunch. When are you back?

**Comment by phelian** [comment]
heading back today, lets sync for lunch sometime next week =)

**Pull request merged**